book/src/02_hello_world.md

```diff
@@ -43,10 +44,30 @@ impl Component for Velocity {
 }
 ```
 
-These will be our components, stored in a `VecStorage`
-(see [the storages chapter][sc] for more details), so we can associate some data
-with an entity. Before doing that, we need to create a world, because this is
-where the storage for all the components is located.
+These will be our two component types. Optionally, `specs` includes a convenient
```

Comment:
I'm not sure. I think we agreed we wouldn't want `#[derive]` shown in the book - except in one dedicated chapter.